research/object_detection/meta_architectures/faster_rcnn_meta_arch.py

```diff
@@ -694,6 +688,8 @@ class subclassing FasterRCNNMetaArch is responsible for any additional
        true_image_shapes) = shape_utils.resize_images_and_return_shapes(
            inputs, self._image_resizer_fn)
 
+      self._resize_shape = resized_inputs.shape.as_list()
```

Comment:
Instead of this, can we pass `image_shape` to `_compute_second_stage_input_feature_maps ` on line 1062 and reuse it ?